src/main/java/frc/robot/RobotContainer.java

```diff
@@ -76,6 +82,7 @@ public RobotContainer()
             case SIM:
                 // Sim robot, instantiate physics sim IO implementations
                 _drive = new Drive(new GyroIO() {}, new ModuleIOSim(), new ModuleIOSim(), new ModuleIOSim(), new ModuleIOSim());
+                _vision = new Vision(_drive, new VisionIO() {}); // Empty implementation for sim
```

Comment:
I've been thinking about this one off and on. PhotonLib works entirely based on NetworkTables, so it should be possible to launch a local version of PhotonVision on a laptop when running the simulator and have it connect to the virtual robot. I _think_ we can use the `VisionIOPhotonLib` class here:

 ```suggestion
                _vision = new Vision(_drive, new VisionIOPhotonLib(_drive));
```